amnestywebsite/humanity-theme
# Register and Render Links with Icons block in PHP

Ref: https://github.com/amnestywebsite/humanity-theme/issues/143

Registers and renders the Links with Icons block in PHP and accounts for block deprecations

**Steps to test**:
1.  Edit a post
2. Check the LWI block appears in the block editor
3. Insert the block, populate with content
4. Save and view the front end

To check deprecation, create the LWI block BEFORE switching to this branch and then switch branch and make sure the block does not break

Example:
http://bigbite.im/i/L8mOXX
http://bigbite.im/i/Z0r7uP
